noetic/distribution.yaml

```diff
@@ -7288,6 +7288,12 @@ repositories:
       url: https://github.com/Jmeyer1292/opw_kinematics.git
       version: master
     status: developed
+  ortools_vendor:
```

Comment:
Do you want source and doc entries here?

Reply:
Added :)